### PR TITLE
Updated docs for the 0.8.0 release: MAISTRA-208

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

File: topics/docs/comparison-with-istio-community.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+:toc:
+
+An installation of Maistra differs from upstream Istio community installations in multiple ways. The modifications to Maistra are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift or OKD.
+
+The current release of Maistra differs from the current upstream Istio community release in the following ways:
+
+Automatic Injection
+-------------------
+
+Enabling automatic injection for your deployments differs between the upstream Istio releases and the Maistra releases.  The upstream sidecar injector will inject all deployments within specifically labelled namespaces whereas the Maistra version relies on opting in and the present of the `sidecar.istio.io/inject` annotation.  For more information please refer to the [the automatic injection section](automatic-injection).
```
Comment (brian-avery) [inline topics/docs/comparison-with-istio-community.adoc]: the presence of

Also, two spaces between inject` and annotation.

File: topics/docs/comparison-with-istio-community.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Role Based Access Control features
+----------------------------------
+
+Role Based Access Control (RBAC) provides a mechanism to control access to a service. You can identify subjects by username or by specifying a set of properties and apply access controls accordingly.
+
+The upstream Istio community installation includes options to perform exact matches on a header, identifying headers which are present (wildcard), or checkin for a header containing a specific prefix or suffix.
```
Comment (brian-avery) [inline topics/docs/comparison-with-istio-community.adoc]: checking for a header

File: topics/docs/comparison-with-istio-community.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  name: httpbin-client-binding
+  namespace: httpbin
+spec:
+  subjects:
+  - user: "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
+    properties:
+      request.headers[<header>]: "value"
+----
+
+The Miastra distribution extends the ability to match request headers by supporting the use of a regular expression. Specify a property key of `request.regex.headers` with a regular expression.
```
Comment (brian-avery) [inline topics/docs/comparison-with-istio-community.adoc]: Wording here is somewhat odd. Is this better?

The Maistra distribution extends the ability to match request headers by supporting the use of  regular expressions. To use this functionality, specify a property key of `request.regex.headers` with a regular expression as the value.

File: topics/docs/comparison-with-istio-community.adoc (hunk 1 above)
Comment (knrc) [reply topics/docs/comparison-with-istio-community.adoc]: aargh and I can't even blame auto completion

Comment (knrc) [reply topics/docs/comparison-with-istio-community.adoc]: There's only one space though, not two.  Did you mean between inject` and annotation?

File: topics/docs/comparison-with-istio-community.adoc (hunk 2 above)
Comment (knrc) [reply topics/docs/comparison-with-istio-community.adoc]: thanks

File: topics/docs/comparison-with-istio-community.adoc (hunk 3 above)
Comment (knrc) [reply topics/docs/comparison-with-istio-community.adoc]: much better

File: topics/docs/comparison-with-istio-community.adoc (hunk 1 above)
Comment (brian-avery) [reply topics/docs/comparison-with-istio-community.adoc]: Lol. Now that I look at that a second, time, I don't see that. But...

annotation.  For

and 

releases.  The

both have double spaces.

Comment (knrc) [reply topics/docs/comparison-with-istio-community.adoc]: I was taught to type on a typewriter and the style was two spaces after a full stop :)

Comment (brian-avery) [reply topics/docs/comparison-with-istio-community.adoc]: Interesting :)

**Pull request merged**